thirdeye-ui/src/app/locale/languages/en-us.json

```diff
@@ -749,6 +749,7 @@
         "select-the-start-and-end-date-time-range-for-the-anomalous-behavior": "Select the start and end date-time range for the anomalous behavior",
         "select-the-top-contributors-to-see-the-dimensions": "Expand/collapse to view the Top Contributors. Select the Top Contributors to see the dimensions plotted in the chart below.",
         "select-who-to-notify-when-finding-anomalies": "Select who to notify when finding anomalies",
+        "send-separate-slack-messages": "Send separate Slack messages",
```

Comment:
I think the copy could be improved. 
Not sure but 

"Send separate Slack messages for each anomaly"

May be clearer. Or maybe explain the alternative. With an info tooltip.
"If not enabled, multiple anomalies may be sent in a single message."

Will let others chime in.